### ISSUE TITLE
Is this project dead? Domain weoffersolution.com Dead for Docs

### ISSUE THREAD

**Issue by JohnRSim** [osadavc/astro-auth#27]
Can't access https://astro-auth.weoffersolution.com/ says domain is up for sale? 

Thanks

**Comment by JulianCataldo** [comment]
You could use deployed env's, like prod. here:

https://astro-auth-docs-mlb49y74m-osada.vercel.app/getting-started/introduction

However I've noticed some newer docs content in the code itself (like some typos fixes).

---

I wonder if some efforts can be joined with the new, agnostic NextAuth, now named "Auth.js".

The wrapper for SvelteKit seems pretty thin:
https://github.com/nextauthjs/next-auth/tree/main/packages/frameworks-sveltekit

Basically, it's a client function with fetchers/forms submissions and a server part with middleware handlers.

**Comment by tajkirkpatrick** [comment]
@JulianCataldo do you know of any communities or maintainers willing to explore that Auth.js and AstroJS collaboration idea. I have a fork repo open, but very little experience contributing to OSS. So I could use the help, I have somewhat of a ball already rolling.
